src/site/content/en/blog/app-shortcuts/index.md

```diff
@@ -0,0 +1,233 @@
+the app shortcut. Examples of app shortcuts include the following:
+
+- Top-level navigation items (e.g., home, timeline, recent orders)
+- Search
+- Data entry tasks (e.g., compose an email or tweet, add a receipt)
+- Activities (e.g., start a chat with the most popular contacts)
+
+{% Aside %}
+Big thanks to the folks at Microsoft Edge and Intel for designing and
+standardizing app shortcuts. Chrome depends on a community of committers working
+together to move the Chromium project forward. Not every Chromium committer is a
+Googler, and these contributors deserve special recognition!
+{% endAside %}
+
+## Define app shortcuts in the web app manifest
+
+App shortcuts are optionally defined in the [web app manifest], a JSON file that
+tells the browser about your Progressive Web App and how it should behave when
+installed on the user's desktop or mobile device. More specifically, they are
+declared in the `shortcuts` array member. Below is an
+example of a potential web app manifest.
+
+```json
+{
+  "name": "Player FM",
+  "start_url": "https://player.fm?utm_source=homescreen",
+  …
+  "shortcuts": [
+    {
+      "name": "Open Play Later",
+      "short_name": "Play Later",
+      "description": "View the list of podcasts you saved for later",
+      "url": "/play-later?utm_source=homescreen",
+      "icons": [{ "src": "/icons/play-later.png", "sizes": "192x192" }]
+    },
+    {
+      "name": "View Subscriptions",
+      "short_name": "Subscriptions",
+      "description": "View the list of podcasts you listen to",
+      "url": "/subscriptions?utm_source=homescreen",
+      "icons": [{ "src": "/icons/subscriptions.png", "sizes": "192x192" }]
+    }
+  ]
+}
+```
+
+Each member of the `shortcuts` array is a dictionary that contains at least a
+`name` and a `url`. Other members are optional.
+
+### name
+
+The human-readable label for the app shortcut when
+displayed to the user.
+
+### short_name (optional)
+
+The human-readable label used where space is limited. It is recommended
+that you provide it, even though it's
+optional.
+
+### description (optional)
+
+The human-readable purpose for the app shortcut.
+It is not used at the time of writing.
+
+### url
+
+The URL opened when a user activates the app
+shortcut. This URL must exist within the scope of the web app manifest. If it is
+a relative URL, the base URL will be the URL of the web app manifest.
+
+### icons (optional)
+
+An array of image resource objects. Each object must
+include the `src` and a `sizes` property. The `type` of image is optional.
+
+SVG files are not supported at the time of writing, use PNG instead.
+
+If you want pixel-perfect icons, provide them in
+increments of 48dp. Otherwise, it is recommended that you use a 192x192 pixels
+icon.
+
+## Test your app shortcuts
+
+To verify your app shortcuts are setup correctly, use the **Manifest** pane in the
+**Application** panel of DevTools.
```

Comment:
Oh, cool!

Also, I missed this new feature in What's New In DevTools (Chrome 84). Do you have cycles to add a section to the post? It's OK if not, but I just figured it doesn't hurt to ask.

Reply:
I just added it two days ago ;)
https://chromiumdash.appspot.com/commit/9fee874360dbc0f891609a8aa1a9f8947b510253
Can you link to the post?

Reply:
https://developers.google.com/web/updates/2020/05/devtools/

Reply:
https://github.com/google/WebFundamentals/pull/8716